src/doc/dev_manual/Spack.rst

```diff
@@ -0,0 +1,362 @@
+        - spec: gawk@4.2.1
+          prefix: /usr
+      gcc:
+        externals:
+        - spec: gcc@7.5.0 languages=c,c++,fortran
+          prefix: /usr
+          extra_attributes:
+            compilers:
+              c: /usr/bin/gcc-7
+              cxx: /usr/bin/g++
+              fortran: /usr/bin/gfortran-7
+      ghostscript:
+        externals:
+        - spec: ghostscript@9.52
+          prefix: /usr
+      git:
+        externals:
+        - spec: git@2.26.2~tcltk
+          prefix: /usr
+      gmake:
+        externals:
+        - spec: gmake@4.2.1
+          prefix: /usr
+      groff:
+        externals:
+        - spec: groff@1.22.3
+          prefix: /usr
+      libtool:
+        externals:
+        - spec: libtool@2.4.6
+          prefix: /usr
+      m4:
+        externals:
+        - spec: m4@1.4.18
+          prefix: /usr
+      ncurses:
+        externals:
+        - spec: ncurses@6.1.20180317+termlib abi=6
+          prefix: /usr
+      openssh:
+        externals:
+        - spec: openssh@8.1p1
+          prefix: /usr
+      openssl:
+        externals:
+        - spec: openssl@1.1.1d
+          prefix: /usr
+      perl:
+        externals:
+        - spec: perl@5.26.1~cpanm+shared+threads
+          prefix: /usr
+      pkg-config:
+        externals:
+        - spec: pkg-config@0.29.2
+          prefix: /usr
+      python:
+        externals:
+        - spec: python@2.7.18+bz2+ctypes~dbm~lzma+nis+pyexpat+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
+          prefix: /usr
+        - spec: python@3.6.13+bz2+ctypes~dbm+lzma+nis+pyexpat+readline+sqlite3+ssl~tix~tkinter+uuid+zlib
+          prefix: /usr
+      rsync:
+        externals:
+        - spec: rsync@3.1.3
+          prefix: /usr
+      ruby:
+        externals:
+        - spec: ruby@2.5.9
+          prefix: /usr
+      sed:
+        externals:
+        - spec: sed@4.4
+          prefix: /usr
+      tar:
+        externals:
+        - spec: tar@1.30
+          prefix: /usr
+      texinfo:
+        externals:
+        - spec: texinfo@6.5
+          prefix: /usr
+      xz:
+        externals:
+        - spec: xz@5.2.3
+          prefix: /usr
+      all:
+        compiler: [gcc, cce]
+        providers:
+          mpi: [cray-mpich]
+      cray-mpich:
+        buildable: false
+        externals:
+        - prefix: /opt/cray/pe/mpich/8.1.10/ofi/gnu/9.1
+          spec: cray-mpich@8.1.10%gcc
+          modules:
+          - cray-mpich/8.1.10
+            cray-pmi/6.0.14
+            cray-pmi-lib/6.0.14
+            libfabric/1.11.0.4.75
+        - prefix: /opt/cray/pe/mpich/8.1.10/ofi/cray/10.0
+          spec: cray-mpich@8.1.10%cce
+          modules:
+          - cray-mpich/8.1.10
+            cray-pmi/6.0.14
+            cray-pmi-lib/6.0.14
+            libfabric/1.11.0.4.75
+
+Debugging a spack package
+-------------------------
+
+When doing a spack install and the install fails, it will automatically keep the stage directory, which will allow you debug the failure.
```

Comment:
Are you using *stage* here for the first time? I might re-word slightly and say...

...it will automatically keep the directory where it did the work, called the *stage* directory, which will allow...